src/ctl/src/cmd_impl/meta/migration.rs

```diff
@@ -0,0 +1,837 @@
+                    prev_id: Set(vd.prev_id as _),
+                    max_committed_epoch: Set(vd.max_committed_epoch as _),
+                    safe_epoch: Set(vd.safe_epoch as _),
+                    trivial_move: Set(vd.trivial_move),
+                    full_version_delta: Set(vd.to_protobuf().into()),
+                })
+                .collect_vec(),
+        )
+        .exec(&meta_store_sql.conn)
+        .await?;
+    }
+    println!("hummock version delta migrated");
+
+    // hummock version stat
+    let version_stats = HummockVersionStats::list(&meta_store)
+        .await?
+        .into_iter()
+        .next();
+    if let Some(version_stats) = version_stats {
+        hummock_version_stats::Entity::insert(hummock_version_stats::ActiveModel {
+            id: Set(version_stats.hummock_version_id as _),
+            stats: Set(TableStats(version_stats.table_stats)),
+        })
+        .exec(&meta_store_sql.conn)
+        .await?;
+    }
+    println!("hummock version stats migrated");
+
+    // compaction
+    // compaction config
+    let compaction_groups = CompactionGroup::list(&meta_store).await?;
+    if !compaction_groups.is_empty() {
+        compaction_config::Entity::insert_many(
+            compaction_groups
+                .into_iter()
+                .map(|cg| compaction_config::ActiveModel {
+                    compaction_group_id: Set(cg.group_id as _),
+                    config: Set((*cg.compaction_config).clone().into()),
+                })
+                .collect_vec(),
+        )
+        .exec(&meta_store_sql.conn)
+        .await?;
+    }
+    println!("compaction config migrated");
+
+    // compaction status
+    let compaction_statuses = CompactStatus::list(&meta_store).await?;
+    if !compaction_statuses.is_empty() {
+        compaction_status::Entity::insert_many(
+            compaction_statuses
+                .into_iter()
+                .map(|cs| compaction_status::ActiveModel {
+                    compaction_group_id: Set(cs.compaction_group_id as _),
+                    status: Set(LevelHandlers(cs.level_handlers.iter().map_into().collect())),
+                })
+                .collect_vec(),
+        )
+        .exec(&meta_store_sql.conn)
+        .await?;
+    }
+    println!("compaction status migrated");
+
+    // compaction task
+    let compaction_tasks = CompactTaskAssignment::list(&meta_store).await?;
+    if !compaction_tasks.is_empty() {
+        compaction_task::Entity::insert_many(compaction_tasks.into_iter().map(|ct| {
+            let context_id = ct.context_id;
+            let task = ct.compact_task.unwrap();
+            compaction_task::ActiveModel {
+                id: Set(task.task_id as _),
+                context_id: Set(context_id as _),
+                task: Set(task.into()),
+            }
+        }))
+        .exec(&meta_store_sql.conn)
+        .await?;
+    }
+    println!("compaction task migrated");
+
+    // hummock sequence
```

Comment:
For ease of future maintenance, may make the init value global constant. Because the init value must be equal to that from `src/meta/src/hummock/manager/sequence.rs`.